src/main/kotlin/com/autonomousapps/tasks/classAnalysisTasks.kt

```diff
@@ -122,7 +122,7 @@ open class ClassListAnalysisTask @Inject constructor(
     }
 
     /**
-     * Class files generated by Kotlin source.
+     * Class files generated by Kotlin source. May be empty.
      */
     @PathSensitive(PathSensitivity.RELATIVE)
```

Comment:
Both inputs should have the @Classpath annotation: https://docs.gradle.org/current/javadoc/org/gradle/api/tasks/Classpath.html

Reply:
Good catch. I'll do a pass through all the tasks and double check normalization in a separate PR.